### ISSUE TITLE
A new structure for the Welcome page

### ISSUE THREAD

**Issue by contraexemplo** [opencollective/documentation-old#32]
## Proposal

This is the current structure of the Welcome page:

```
Welcome
├── Introductory paragraphs
│   ├── Short description of Open Collective
│   ├── Short description of the goal of our documentation
│   ├── Short instructions to suggest changes to any page
│   └── Short instructions to contact us: email and Slack
├── Overall FAQ
└── How to contribute to these docs
```

I propose restructuring the page as follows:
```
Welcome
├── Introductory paragraphs
│   ├── Short description of Open Collective
│   ├── Short description of the goal of our documentation
│   ├── How to explore our documentation
│           ├── Map explaining every section
│           └── Instructions on using the search bar
│   ├── How to contribute to our documentation
│           └──  Points to Contributing > Documentation > Suggesting changes
│   └── Short instructions to contact us: email and Slack
└── Overall FAQ
```

## Documentation map
It should be added after revamping **all** pages of the documentation. Proposed format:
```
Name of the section
└── Brief description + link
```

## Contribution guidelines
A draft of the **Suggesting changes** page is available [on my own GitBook](https://documentarians.gitbook.io/metadocumentation/open-collective/contribution-guidelines/documentation/using-gitbook). It still needs:
- [x] Higher quality screenshots to add complementary information to every instruction
- [x] Instructions to add new files (media) or pages and sections to the repository

**Comment by alanna** [comment]
This looks good! Seems like a good approach. We want people to be able to quickly find what they are looking for and get an overview.

**Comment by contraexemplo** [comment]
Closed by 57a274450fce45f54ec5c606a4444d7b15ee96d3.
